src/main/java/com/matchbook/sdk/clients/rest/dtos/user/LoginResponse.java

```diff
@@ -1,11 +1,45 @@
 package com.matchbook.sdk.clients.rest.dtos.user;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.matchbook.sdk.clients.rest.dtos.MatchbookResponse;
 
+@JsonIgnoreProperties(ignoreUnknown = true)
 public class LoginResponse implements MatchbookResponse {
 
     private String sessionToken;
-    private Long userId;
+    private long userId;
     private Account account;
 
+    public String getSessionToken() {
+        return sessionToken;
+    }
+
+    public void setSessionToken(String sessionToken) {
+        this.sessionToken = sessionToken;
+    }
+
+    public long getUserId() {
+        return userId;
+    }
+
+    public void setUserId(Long userId) {
+        this.userId = userId;
+    }
+
+    public Account getAccount() {
+        return account;
+    }
+
+    public void setAccount(Account account) {
+        this.account = account;
+    }
+
+    @Override
+    public String toString() {
+        return "LoginResponse{" +
```

Comment:
We should agree on a `toString()` standard format :grin:

Reply:
What's wrong with this one? :D

Reply:
Single vs double quotes, chars vs String usage, quotation for String attributes... :innocent: 
It's a matter of style. I simply don't like this style too much.